src/main/resources/application.yaml

```diff
@@ -23,9 +23,8 @@ idam:
 
 management:
   info.git.mode: full
-  endpoints:
-    health.show-details: always
-    web.base-path: /
+  endpoints.web.base-path: /
+  endpoint.health.show-details: always
```

Comment:
application.yaml: Is this an intentional change? You've gone from "endpoints.health.show-details" to "endpoint.health.show-details" (lost an 's')

Reply:
Yes, it was intentional. The /health is not showing all details because of that typo. This change will fix it.